organisaatio-ui/src/types/types.ts

```diff
@@ -78,8 +78,8 @@ export type Yhteystiedot = YhteystiedotEmail | YhteystiedotPhone | YhteystiedotW
 export interface Organisaatio {
     oid?: string;
     ytunnus: string;
-    nimi: any;
-    nimet: any;
+    nimi: any; // TODO?
```

Comment:
TODO

Reply:
added todo because nimi handling is a bit of a mystery to me and will check it later.